diff --git a/src/os_comet.eliom b/src/os_comet.eliom
index 8e6ed39..a64130d 100644
--- a/src/os_comet.eliom
+++ b/src/os_comet.eliom
@@ -85,8 +85,10 @@ let already_send_ref =
 
 let%client handle_error =
   ref (fun exn ->
-    Eliom_lib.Lwt_log.ign_info_f ~exn
-      "Exception received on Os_comet's monitor channel: ";
+    Logs.info (fun fmt ->
+      fmt
+        "Exception received on Os_comet's monitor channel: %s"
+	(Printexc.to_string exn));
     restart_process ();
     Lwt.return_unit)
 
@@ -95,7 +97,7 @@ let%client set_error_handler f = handle_error := f
 let%client handle_message = function
   | Error exn -> !handle_error exn
   | Ok Heartbeat ->
-      Eliom_lib.Lwt_log.ign_info_f "poum";
+      Logs.info (fun fmt -> fmt "poum");
       Lwt.return_unit
   | Ok Connection_changed ->
       Os_msg.msg ~level:`Err
diff --git a/src/os_msg.eliom b/src/os_msg.eliom
index 69023eb..901c8f2 100644
--- a/src/os_msg.eliom
+++ b/src/os_msg.eliom
@@ -46,7 +46,7 @@ let%shared msg ?(level = `Err) ?(duration = ~%(!default_duration))
            if ~%onload then Eliom_client.lwt_onload () else Lwt.return_unit
          in
          let msgbox = msgbox () in
-         Eliom_lib.Lwt_log.ign_info_f "%s" ~%message;
+         Logs.info (fun fmt -> fmt "%s" ~%message);
          Dom.appendChild msgbox message_dom;
          let* () = Js_of_ocaml_lwt.Lwt_js.sleep ~%duration in
          Dom.removeChild msgbox message_dom;
diff --git a/src/os_session.eliom b/src/os_session.eliom
index 29b4055..65f4663 100644
--- a/src/os_session.eliom
+++ b/src/os_session.eliom
@@ -20,7 +20,7 @@
 
 open Lwt.Syntax
 
-let log_section = Lwt_log.Section.make "os:session"
+let log_section = Logs.Src.create "os:session"
 let user_indep_state_hierarchy = Eliom_common.create_scope_hierarchy "userindep"
 let user_indep_process_scope = `Client_process user_indep_state_hierarchy
 let user_indep_session_scope = `Session user_indep_state_hierarchy
@@ -41,7 +41,8 @@ let mk_action_queue name =
           let* () = oldf arg in
           f arg)
   , fun arg ->
-      Lwt_log.ign_debug ~section:log_section ("handling actions: " ^ name);
+      Logs.debug ~src:log_section (fun fmt ->
+        fmt "handling actions: %s" name);
       !r arg )
 
 let on_connected_request, connected_request_action =
diff --git a/src/os_user_view.eliom b/src/os_user_view.eliom
index 60f10a4..dec2ad3 100644
--- a/src/os_user_view.eliom
+++ b/src/os_user_view.eliom
@@ -295,7 +295,8 @@ let%shared upload_pic_link ?(a = []) ?(content = [txt "Change profile picture"])
                      Lwt.return_unit)
                   (fun exn ->
                      Os_msg.msg ~level:`Err "Error while uploading the picture";
-                     Eliom_lib.Lwt_log.ign_info_f "%s" ~exn "→ ";
+                     Logs.info (fun fmt ->
+		       fmt "→ %s" (Printexc.to_string exn));
                      Lwt.return_unit))
             : _)]
       :: a)
